lib.ftd

```diff
@@ -3507,7 +3495,7 @@ width.fixed.px: 19
 -- end: ftd.row
 
 -- ftd.row:
-if: { ftd.dark-mode && !ftd.follow-system-dark-mode }
+if: { ftd.dark-mode }
```

Comment:
@MeenuKumari28 plz refer this: https://github.com/fastn-community/color-doc/blob/f9f945d89449874c2475353bc325a8fb07afdf86/index.ftd#L611